code/gradle.properties

```diff
@@ -21,4 +21,4 @@ displayName = Image lightbox
 vendorName = Bouvet Norge AS
 vendorUrl = http://www.bouvet.no
 xpVersion = 7.9.0
-version = 1.1.0
+version = 2.0.0
```

Comment:
Perhaps bumping to version 1.1.1 or 1.2.0 might be more accurate as the release is mostly going to be just bugfixes, package updates and small functionality improvements. Or you did choose to go for 2.0.0 for a specific reason? :)

Reply:
Chose to bump to 2.0.0 cause the xp version is updated and @tunestam said that could lead to breaking changes 😊